code/game/objects/items/devices/scanners.dm

```diff
@@ -402,4 +393,94 @@ REAGENT SCANNER
 
 	var/value = get_value(target)
 	user.visible_message("\The [user] scans \the [target] with \the [src]")
-	user.show_message("Price estimation of \the [target]: [value ? value : "N/A"] Credits")
\ No newline at end of file
+	user.show_message("Price estimation of \the [target]: [value ? value : "N/A"] Credits")
+
+
+/obj/item/device/breath_analyzer
+	name = "breath analyzer"
+	desc = "A hand-held breath analyzer that provides a robust amount of information about the subject's respitory system."
+	icon_state = "breath_analyzer"
+	item_state = "analyzer"
+	w_class = 2.0
+	flags = CONDUCT
+	slot_flags = SLOT_BELT
+	throwforce = 5
+	throw_speed = 4
+	throw_range = 20
+	matter = list(DEFAULT_WALL_MATERIAL = 30,"glass" = 20)
+	origin_tech = list(TECH_MAGNET = 2, TECH_BIO = 2)
+
+/obj/item/device/breath_analyzer/attack(mob/living/M as mob, mob/living/user as mob)
+
+	var/mob/living/carbon/human/H = M
+
+	if (!istype(H))
+		user.show_message("<span class='warning'>You can't find a way to use the [src] on [M].</span>")
+		return
+
+	if ( ((CLUMSY in user.mutations) || (DUMB in user.mutations)) && prob(20))
+		user.show_message("<span class='danger'>Your hand slips from clumsiness!</span>")
+		eyestab(M,user)
+		user.show_message("<span class='danger'>Alert: No breathing detected.</span>")
+		return
+
+	if (!user.IsAdvancedToolUser())
+		user.show_message("<span class='warning'>You don't have the dexterity to do this!</span>")
+		return
+
+	if(user == H && !H.can_eat(src))
+		return
+	else if(!H.can_force_feed(user, src))

```

Comment:
No messages for these?

Reply:
Messages are handled inside the procs can_eat and can_force_feed.